File: 프로그래머스/0/120890. 가까운 수/가까운 수.java
import java.util.*; 
class Solution {
    public int solution(int[] array, int n) {
        Arrays.sort(array) ; 
        int result = 1000000000;  
        int answer = 0 ; 
        for(int num : array){
            int range = Math.abs(num-n) ;
            if(result > range) {
                result = range ; 
                answer = num ; 
                        
            }
        }
        return answer ;       
    }
}

File: 프로그래머스/0/120890. 가까운 수/가까운 수.js
function solution(array, n) {
    var answer = 100 ;
    console.log(array.sort((a,b) => a-b) );
    let result = 0 ; 
    for(let num of array){
        let min = Math.abs(num - n) ;
        if(answer > min){
            answer = min ; 
            result = num ;
        }
    }
    return result;
}